cdap-gateway/src/test/java/co/cask/cdap/gateway/handlers/log/LogHandlerTestRun.java

```diff
@@ -346,6 +346,24 @@ public void testNonExistenceRunLogs() throws Exception {
     getLogs(MockLogReader.TEST_NAMESPACE, MockLogReader.SOME_WORKFLOW_APP.getApplication(), "workflows",
             MockLogReader.SOME_WORKFLOW, RunIds.generate().getId(), "prev", HttpResponseStatus.NOT_FOUND.code());
   }
+
+  @Test
+  public void testFilterJson() throws Exception {
```

Comment:
either add comment in test case, or rename method name to be more specific, `testFilterWithEmptyResult`